addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -292,11 +292,12 @@
                         <option t-esc="record.display_name" t-att-value="record.id" t-att="{selected: record.selected and 'selected' or None}"/>
                     </t>
                 </select>
-                <div id="editable_select" class="form-control s_website_form_input">
-                    <t t-foreach="field.records" t-as="record" t-key="record_index">
-                        <t t-set="noValueLabel">no value</t>
-                        <div t-esc="record.display_name" t-attf-data-empty-value="&lt;#{noValueLabel}&gt;" t-att-id="record.id" t-attf-class="s_website_form_select_item #{record.selected and 'selected' or ''}"/>
-                    </t>
+                <div id="editable_select" class="s_website_form_input">
+                    <select class="form-select">
+                        <t t-foreach="field.records" t-as="record" t-key="record_index">
+                            <option t-if="record.selected" t-att-id="field.id + record_index" t-esc="record.display_name" t-att-value="record.id" selected="selected"/>
+                        </t>
+                    </select>
```

Comment:
Given that we now display a `<select>` anyway, can't we just get rid of the fake `editable_select` one ? (and only keep the real select)

Reply:
@bso-odoo Good idea. I'm gonna test that.